extensions/stackdriver/metric/record.cc

```diff
@@ -244,29 +244,82 @@ void addHttpSpecificTags(const ::Wasm::Common::RequestInfo& request_info,
                        std::to_string(response_code));
 }
 
+TagKeyValueList getMetricTagMap(
+    TagKeyValueList input_map,
+    std::vector<std::pair<std::string, std::string>> overrides) {
```

Comment:
I feel `const&` is good for the two args input_map and overrides

Reply:
done.